src/containers/About/About.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import "./About.css";
+import aboutImage from "./aboutImage.png";
+import { Row } from "react-bootstrap";
+
+const About = () => {
+  return (
+    <div className="aboutUsMainDiv">
```

Comment:
please move this content into a component and then render it here in the conainer, usually in a container u dont want to write any content u want to render the components right away here

Reply:
Noted, im on it